### DISCUSSION TITLE
Help Needed with Controller and Route Setup

### DISCUSSION THREAD

**Discussion by rentaolulin** [Laravel-Backpack/community-forum#1098] [Q&A (Help)] [answered]
Hello everyone,

I'm having some trouble with my Backpack for Laravel setup and could really use some assistance.

I have a ProfileCrudController and within this controller, I have a method named drafts which contains the following line of code:

$this->setupListOperation();
$this->crud->addClause('where', 'status', 0);

I have two menu items:

One with the link backpack_url('profile')
Another with the link backpack_url('profile/drafts')

My route settings are as follows:

Route::crud('profile', 'ProfileCrudController');
Route::get('profile/drafts', 'ProfileCrudController@drafts');

When I navigate to the profile URL, it works perfectly. However, when I click on the profile/drafts link, the page is completely blank with no content displayed.

Does anyone have any idea what might be going wrong or how I can fix this issue? Any help would be greatly appreciated!

Thank you in advance.

**Comment by karandatwani92** [comment, marked as answer]
Hey @rentaolulin 

It seems you want to show only draft items. I recommend the following ways to achieve this.
- [Setup Custom Views for ListOperation](https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#custom-views-for-listoperation-pro)
- Setup a simple [filter](https://backpackforlaravel.com/docs/6.x/crud-filters#filter-types-1).

**Reply by rentaolulin** [reply]
Thank you so much !
